satpy/readers/viirs_l2.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2011-2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Interface to VIIRS L2 files."""
+
+import logging
+from datetime import datetime
+
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
+
+LOG = logging.getLogger(__name__)
```

Comment:
I think lowercase, for example, `logger`, would fit better with style guides.

Reply:
The logging wasn't even used, so removed it in 0720a1f05